go-controller/pkg/node/gateway_localnet.go

```diff
@@ -116,32 +122,107 @@ func generateGatewayNATRules(ifname string, ip net.IP) []iptRule {
 	return rules
 }
 
-func localnetGatewayNAT(ipt util.IPTablesHelper, ifname string, ip net.IP) error {
-	rules := generateGatewayNATRules(ifname, ip)
-	return addIptRules(ipt, rules)
+func localnetGatewayNAT(localNetdata []*localnetData, ifname string) error {
+	for _, lnData := range localNetdata {
+		rules := generateGatewayNATRules(ifname, lnData.gatewayIP)
+		err := addIptRules(lnData.ipt, rules)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
 }
 
-func initLocalnetGateway(nodeName string, subnet *net.IPNet, wf *factory.WatchFactory, nodeAnnotator kube.Annotator) error {
-	// Create a localnet OVS bridge.
+func initLocalnetGateway(nodeName string, subnets []*net.IPNet, wf *factory.WatchFactory, nodeAnnotator kube.Annotator) error {
+
 	localnetBridgeName := "br-local"
-	_, stderr, err := util.RunOVSVsctl("--may-exist", "add-br",
-		localnetBridgeName)
+
+	// Evaluate input subnets
+	err := evaluateInputNetworks(subnets)
 	if err != nil {
-		return fmt.Errorf("Failed to create localnet bridge %s"+
-			", stderr:%s (%v)", localnetBridgeName, stderr, err)
+		return err
 	}
 
-	ifaceID, macAddress, err := bridgedGatewayNodeSetup(nodeName, localnetBridgeName, localnetBridgeName, true)
+	// Setup local brifge and ports
+	ifaceID, macAddress, link, err := createAndSetupLocalBridgePorts(localnetBridgeName, nodeName)
```

Comment:
again, comment doesn't add anything. Basically, most of the added comments should be removed

Reply:
The comment still has spelling errors, and isn't needed. I don't think this change is adding dual-stack support. If this is clean up then move it to its own commit.